wizard.py

```
@@ -259,12 +276,20 @@ def run_service_setup(service_name, selected_services, https_enabled=False, serv
             
     except FileNotFoundError as e:
         console.print(f"❌ {service_name} setup failed: {e}")
+        console.print(f"[yellow]   Check that the service directory exists: {service['path']}[/yellow]")
+        console.print(f"[yellow]   And that 'uv' is installed and on your PATH[/yellow]")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)